frontend/src/packages/projects/projects-api.ts

```diff
@@ -0,0 +1,34 @@
+import { type ProjectGetAllResponseDto } from 'shared/build/index.js';
+import { ProjectsApiPath } from 'shared/build/index.js';
+
+import { ApiPath, ContentType } from '~/libs/enums/enums.js';
+import { HttpApi } from '~/libs/packages/api/api.js';
+import { type IHttp } from '~/libs/packages/http/http.js';
+import { type IStorage } from '~/libs/packages/storage/storage.js';
+
+type Constructor = {
+  baseUrl: string;
+  http: IHttp;
+  storage: IStorage;
+};
+
+class ProjectsApi extends HttpApi {
+  public constructor({ baseUrl, http, storage }: Constructor) {
+    super({ path: ApiPath.PROJECTS, baseUrl, http, storage });
+  }
+
+  public async getUserProjects(): Promise<ProjectGetAllResponseDto> {
+    const response = await this.load(
+      this.getFullEndpoint(ProjectsApiPath.ROOT, {}),
```

Comment:
Your current endpoint expects /:userId as parameter, but you didn't specify it

Reply:
should be root